streams/src/main/java/org/apache/kafka/streams/processor/ProcessorContext.java

```diff
@@ -115,30 +115,35 @@
     void commit();
 
     /**
-     * Returns the topic name of the current input record
+     * Returns the topic name of the current input record; could be null if it is not available (for example, it is from the punctuate call)
```

Comment:
nit: "it" is unclear in "for example, it is from". Should probably be something more like "for example, if this method is invoked from". Same for other uses of "it" in doc additions in this file.